extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -75,7 +79,7 @@
     private ThingRegistry thingRegistry;
     private ItemRegistry itemRegistry;
 
-    private static final String DEFAULT_TIME_ZONE_ID = "Europe/Helsinki";
+    private static final ZoneId DEFAULT_TIME_ZONE_ID = ZoneId.of("Europe/Bucharest");
```

Comment:
Where is this constant used?

Reply:
I use it to set new default time zone and locale, which will be used during the tests execution.

Reply:
Of course, my bad... I was hit by the surprise that you could change the variable type without another change in `setUpClass()` and I obviously missed it there. All good!